Src/AutoFixtureUnitTest/DomainNameTest.cs

```diff
@@ -0,0 +1,136 @@
+﻿using Ploeh.AutoFixture;
+using System;
+using Xunit;
+
+namespace Ploeh.AutoFixtureUnitTest
+{
+    public class DomainNameTest
+    {
+        [Fact]
+        public void InitializeWithNullDomainNameThrows()
+        {
+            // Fixture setup
+            // Exercise system and verify outcome
+            Assert.Throws<ArgumentNullException>(
+                () => new DomainName(null));
+            // Teardown
+        }
+
+        [Fact]
+        public void InitializeWithEmptyDomainNameThrows()
+        {
+            // Fixture setup
+            Assert.Throws<ArgumentException>(
+                () => new DomainName(string.Empty));
+            // Exercise system and verify outcome
+            // Teardown
+        }
+
+        [Fact]
+        public void ToStringReturnsCorrectResult()
+        {
+            // Fixture setup
+            var expected = Guid.NewGuid().ToString();
+            var sut = new DomainName(expected);
+            // Exercise system
+            var result = sut.ToString();
+            // Verify outcome
+            Assert.Equal(expected, result);
+            // Teardown
+        }
+
+        [Fact]
+        public void SutDoesNotEqualNullObject()
+        {
+            // Fixture setup
+            var sut = new DomainName(Guid.NewGuid().ToString());
+            object other = null;
+            // Exercise system
+            bool result = sut.Equals(other);
+            // Verify outcome
+            Assert.False(result);
+            // Teardown
+        }
+
+        [Fact]
+        public void SutDoesNotEqualNullSut()
```

Comment:
No, indeed, same path. I've added a commit to remove this superfluous unit test, as well as the other one.

I've also taken the liberty to remove them from the `EmailAddressLocalPart` unit test in a separate PR (#474).